### ISSUE TITLE
Error in SecurityServiceProvider after the last update.

### ISSUE THREAD

**Issue by iBet7o** [silexphp/Silex-Skeleton#42]
This error occurs after the last update.
```
ContextErrorException in ContextListener.php line 42:

Catchable Fatal Error: Argument 1 passed to Symfony\Component\Security\Http\Firewall\ContextListener::__construct() must be an instance of Symfony\Component\Security\Core\SecurityContextInterface, instance of Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorage given, called in /server/vendor/silex/silex/src/Silex/Provider/SecurityServiceProvider.php on line 359 and defined
```

Do you need to update the configuration of the provider?

**Comment by mTorres** [comment]
Sorry, I can't reproduce this...
```bash
    $ composer create-project fabpot/silex-skeleton silex-42.local ~2.0@dev
```
Then adding the SecurityProvider in *app.php*
```php
$app->register(new Silex\Provider\SecurityServiceProvider(),
[
  "security.firewalls" => [
    "secured" => [
      "pattern" => "^/secured$",
      "http" => true,
      "users" => [
        "admin" => ["ROLE_ADMIN", "5FZ2Z8QIkA7UTZ4BYkoC+GsReLf569mSKDsfods6LYQ8t+a8EW9oaircfMpmaLbPBh4FOBiiFyLfuZmTSUwzZg=="],
      ]
    ]
  ]
]);
```
Finally adding a controller for the "/secured" path in *controllers.php*
```php
$app->get('/secured', function() {
  $html = <<<EOH
<!doctype html>
<html>
<head>
<title>Secured!</title>
</head>
<body>
  Sorry man, this page is secured!
</body>
</html>
EOH;

  return new Response($html);
})
->bind('secured')
;
```
I've tried this in master (2.0@dev) as well in 1.1 (1.1 is pulling Silex 1.2.4 right now), so which versions of skeleton and Silex are you using?

**Comment by fabpot** [comment]
should be fixed in silexphp/Silex@c207787

**Comment by iBet7o** [comment]
it works now!

thanks @fabpot :D
